### PR TITLE
Text changes in email and confirmation page

### PR DESCRIPTION
**Pull request by LayCraft**

### CODE REVIEW

**Comment by LayCraft** [comment]
I changed BE and FE htmls.

File: embc-app/Services/Registrations/NewRegistrationService.cs [hunk 1]
```diff
@@ -38,11 +50,7 @@ private EmailMessage CreateEmailMessageForRegistration(ViewModels.Registration r
...
-<br/><br/>
-Your Emergency Support Services File Number is: <b>{registration.EssFileNumber}</b>
+This email has been generated by the Emergency Support Services program to provide you with a record of your Emergency Support Services File Number. If you have not registered online via the {essRegistrationLink} and are receiving this email, please contact your family members to ensure they have not registered on your behalf. If you have received this email in error, please disregard. Your Emergency Support Services File Number is: <b>{registration.EssFileNumber}</b> - If you are under order and require food, clothing, lodging, transportation, incidentals or other emergency supports, proceed to your nearest Reception Centre. A list of open Reception Centres can be found at Emergency Info BC. - If you do not require supports, or are under alert, no further actions are required. - If you are in a Reception Centre, proceed to one of the Emergency Support Services volunteers on site who will be able to assist you with completing your registration. - Please bring your Emergency Support Services File Number with you to the Reception Centre.
```
Comment (severinbeauvais) [inline embc-app/Services/Registrations/NewRegistrationService.cs]: 1. missing link to Emergency Info BC
2. I think the text in the ticket is malformed -- it looks like some of the text is list items

File: embc-app/Services/Registrations/NewRegistrationService.cs [hunk 2]
```diff
@@ -52,21 +60,28 @@ private EmailMessage CreateEmailMessageForRegistration(ViewModels.Registration r
             {
                 body += $@"
 <br/><br/>
-- If you are under order and require food, clothing, lodging, transportation, incidentals or other emergency supports, proceed to your nearest Reception Centre.
-A list of open Reception Centres can be found at {emergencyInfoBCLink}.<br/>
-- If you do not require supports, or are under alert, no further actions are required.<br/>
-- If you are in a Reception Centre, proceed to one of the Emergency Support Services volunteers on site who will be able to assist you with completing your registration.<br/>
-- Your Emergency Support Services File Number has been emailed to you.<br/>
+- If you are under order and require any emergency support services, proceed to your designated Reception Centre. A list of open Reception Centres can be found at {emergencyInfoBCLink}.<br/>
```
Comment (severinbeauvais) [inline embc-app/Services/Registrations/NewRegistrationService.cs]: Duplicate of text above. Please resolve.
Also text below.

File: embc-app/Services/Registrations/NewRegistrationService.cs (hunk 1 above)
Comment (severinbeauvais) [reply embc-app/Services/Registrations/NewRegistrationService.cs]: AMB confirmed that ticket was "wording fix" not "structural fix".

**Pull request merged**